### PR TITLE
GUI-1659 updated beaker to at least 1.7.0 and added pycryptopp depend…

### PR DESCRIPTION
**Pull request by dkavanagh**
…ency

https://eucalyptus.atlassian.net/browse/GUI-1659

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -130,6 +130,7 @@ def make_release_tree(self, base_dir, files):
     'M2Crypto >= 0.20.2',
     'markupsafe >= 0.9.2',
     'pycrypto >= 2.0.1',
+    'pycryptopp >= 0.7.1',
```
Comment (kamalgill) [inline setup.py]: @dkavanagh: I don't think we need pycrypto if we're using pycryptopp, per https://github.com/bbangert/beaker/blob/master/beaker/crypto/pycrypto.py

Comment (dkavanagh) [inline setup.py]: is Crypto being imported, pycrypto?

Comment (kamalgill) [inline setup.py]: Yes, AFAICT.  It's probably harmless to leave pycrypto there since Beaker falls back to pycrypto only when pycryptopp isn't there.

**Comment by kamalgill** [comment]
Supplanted by #1928

**Pull request closed without merging**